testcases/file_as_struct/output.txt

```diff
@@ -0,0 +1,2 @@
+./struct.zig:1:1: found top level fields, file name should be capitalized
+./NotStruct.zig:1:1: found no top level fields, file name should be lowercase
```

Comment:
Should there be a test that the rule doesn't find fault with correctly-capitalized files?